github.sh

```diff
@@ -8,5 +8,5 @@ fi
 
 organization=$1
 
-# JQ
-gh api --paginate "orgs/$organization/repos" | jq -r 'first(["cloneUrl","branch","org"]), (.[] | [.clone_url, .default_branch, "'"$organization"'"]) | @csv'
+echo "\"cloneUrl\",\"branch\",\"org\""
+gh api --paginate "orgs/$organization/repos" --jq '.[] | select(.archived == false) | [.clone_url, .default_branch, "'"$organization"'"] | @csv' | sort
```

Comment:
Figured add the drop in replacement for ssh as a commented out option.
```suggestion
gh api --paginate "orgs/$organization/repos" --jq '.[] | select(.archived == false) | [.clone_url, .default_branch, "'"$organization"'"] | @csv' | sort
#gh api --paginate "orgs/$organization/repos" --jq '.[] | select(.archived == false) | [.ssh_url, .default_branch, "'"$organization"'"] | @csv' | sort
```